docs/mappings/moqo_gbfs_2.3_mapping.md

```diff
@@ -0,0 +1,199 @@
+`eco_label` | -
+`max_range_meters` | Not availabe via API, set to `250.000`
+`name` | `row['car_model_name']`
+`g_CO2_km` | -
+`vehicle_image` | -
+`make` | first word of `row['car_model_name']`
+`model` | all words after the first word of `row['car_model_name']`
+`wheel_count` | -
+`max_permitted_speed` | -
+`rated_power` | -
+`default_reserve_time` | -
+`return_constraint`| `roundtrip`
+`vehicle_assets`| -
+`default_pricing_plan_id`| `tarif` (current provided only has one pricing plan, might be overwritten in subclasses)
+`pricing_plan_ids`| -
+
+
+### station_information.json
+
+Station information is extracted from `parkings` endpoint.
+
+
+#### Field Mappings
+
+GBFS Field | Mapping
+--- | ---
+`station_id` | `row['id']`
+`name` | `row['name']`
+`short_name` | -
+`lat` | `row['center']['lat']`
+`lon` | `row['center']['lon']`
+`address` | `row['town'] + ', ' + row['street'] + ' ' + row['street_number']`
+`cross_street` | -
+`region_id` | -
+`post_code` |  `row['zipcode']`
+`rental_methods` |`['key']`
+`is_virtual_station`| -
+`station_area` | -
+`parking_type` | -
+`parking_hoop` | -
+`contact_phone` | -
+`capacity` | -
+`vehicle_capacity`  |  `row['capacity_max']`
+`vehicle_type_capacity` | _deduced from vehicles assigned to this station free_bike_status.station_id_
+`is_valet_station`  | -
+`is_charging_station` |  -
+`rental_uris` | -
+
+
+### station_status.json
+
+*Describe endpoint from which information is extracted and potential filter rules to be applied*
+
+#### Field Mappings
+
+GBFS Field | Mapping
+--- | ---
+`station_id` | `row['lon']_row['lat']`
+`num_bikes_available` | _deduced by x2gbfs from free_bike_status.station_id_
+`vehicle_types_available` | _deduced by x2gbfs from free_bike_status.station_id_
+`num_docks_available` | -
+`vehicle_docks_available` | -
+`num_docks_disabled` | -
+`is_installed` | `True`
+`is_renting` | `True` (Note: the API does not return opening hours information yet)
+`is_returning` | `True` (Note: the API does not return opening hours information yet)
+`last_reported` | `row['UTC Timestamp']`
+
+
+### free_bike_status.json
+
+Free vehicles are extracted from `cars` endpoint. As currently unavailable cars have no `latest_parking` set, we filter them out. We request availablity status as being available in the next 3 hours.
+
+#### Field Mappings
+
+GBFS Field | Mapping
+--- | ---
+`bike_id` | `row['id']`
+`lat` | - Vehicles are at station, so we don't provide coords
+`lon` | - Vehicles are at station, so we don't provide coords
+`is_reserved` | `row['available'] is not True`
+`is_disabled` | -
+`rental_uris` | -
+`vehicle_type_id` | see section vehicle_types
+`last_reported` | curent time
+`license_plate`: `row['license'].split('|')[0].strip()` (at least Stadtwerk Tauberfranken seems to append station name after the license plate)
+`current_range_meters` | `vehicle_type['max_rang_meters'] * vehicle['fuel_level'] / 100.0`
+`current_fuel_percent` | `vehicle['fuel_level'] / 100.0`
```

Comment:
fuel_level has values <= 100, so it is already a percentage number

Reply:
In GBFS, current_fuel_percent is a float between 0 and 1.0, not an int between 0 and 100.